form-validation.js

```diff
@@ -0,0 +1,55 @@
+const form = document.getElementById('form');
```

Comment:
- [ ] Since you are using Javascript for this project I think it is required to set together The [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in your project repository.  ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: